.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,77 @@
+---
+name: Support
+about: If you have questions about medusa-operator
+title: ''
+labels: question, needs-triage
+assignees: ''
+
+---
+
+<!--
+Thanks for filing an issue! Before hitting the button, please answer these questions.
+
+Fill in as much of the template below as you can.
+
+Note that this repository is about medusa-operator itself. If you have questions specifically about Medusa, please visit https://github.com/thelastpickle/cassandra-medusa.
+
+We will try our best to answer the question, but we also have a mailing list (k8ssandra-users@googlegroups.com.) for any other questions.
+-->
+
+**Type of question**
+<!-- Uncomment one or more of the following lines depending on what you are asking about: -->
```

Comment:
Could this be a checkbox?

Reply:
Cleaning this up now... they were checkboxes, just the comment didn't reflect that.  Updated in my latest commit with the typo fixes.